libs/langchain-community/src/embeddings/voyage.ts

```diff
@@ -41,7 +54,7 @@ export class VoyageEmbeddings
   extends Embeddings
   implements VoyageEmbeddingsParams
 {
-  modelName = "voyage-01";
+  modelName = "voyage-2";
```

Comment:
This is technically breaking - people have complained about this before. Can you revert?

Reply:
How can I accept your change ? ^^